packages/modules/src/modules/banners/designableBanner/DesignableBanner.tsx

```diff
@@ -0,0 +1,324 @@
+import React from 'react';
+import { css } from '@emotion/react';
+import { brand, neutral, space, specialReport } from '@guardian/src-foundations';
+import { BannerEnrichedReminderCta, BannerRenderProps } from '../common/types';
+import { DesignableBannerHeader } from './components/DesignableBannerHeader';
+import { DesignableBannerArticleCount } from './components/DesignableBannerArticleCount';
+import { DesignableBannerBody } from './components/DesignableBannerBody';
+import { DesignableBannerCtas } from './components/DesignableBannerCtas';
+import { DesignableBannerCloseButton } from './components/DesignableBannerCloseButton';
+import { DesignableBannerVisual } from './components/DesignableBannerVisual';
+import { between, from, until } from '@guardian/src-foundations/mq';
+import { SecondaryCtaType } from '@sdc/shared/types';
+import { DesignableBannerReminder } from './components/DesignableBannerReminder';
+import DesignableBannerTicker from './components/DesignableBannerTicker';
+import { templateSpacing } from './styles/templateStyles';
+import useReminder from '../../../hooks/useReminder';
+import useMediaQuery from '../../../hooks/useMediaQuery';
+import useChoiceCards from '../../../hooks/useChoiceCards';
+import { ChoiceCards } from '../choiceCardsButtonsBanner/components/ChoiceCards';
+import { buttonStyles } from './styles/buttonStyles';
+import { BannerTemplateSettings } from './settings';
+import { bannerWrapper, validatedBannerWrapper } from '../common/BannerWrapper';
+
+const DesignableBanner: React.FC<BannerRenderProps> = ({
+    content,
+    onCloseClick,
+    numArticles,
+    onCtaClick,
+    onSecondaryCtaClick,
+    reminderTracking,
+    separateArticleCount,
+    tickerSettings,
+    choiceCardAmounts,
+    countryCode,
+    submitComponentEvent,
+    tracking,
+    design,
+}: BannerRenderProps): JSX.Element => {
+    // We can't render anything without a design
+    if (!design) {
+        return <></>;
+    }
+
+    const templateSettings: BannerTemplateSettings = {
```

Comment:
In future PRs this hard-coded/static configuration will be replaced by configuration read from the `design` prop.